src/utils/components/DiskModal/components/DiskInterfaceSelect/utils/util.ts

```diff
@@ -0,0 +1,15 @@
+import { diskTypes } from '@kubevirt-utils/resources/vm/utils/disk/constants';
+import { getDiskDrive } from '@kubevirt-utils/resources/vm/utils/disk/selectors';
+
+export const getDiskTypeHelperText = (disk: any, isVMRunning: boolean): string => {
+  if (getDiskDrive(disk) === diskTypes.cdrom)
+    return 'CD-ROM type is automatically set and cannot be changed';
+  if (isVMRunning) return 'Hot plug is enabled only for Disk and LUN types';
```

Comment:
Could you please add an empty line between these if statements and those in the function below?